docs/sources/integrations/zendesk/index.md

```diff
@@ -18,13 +18,41 @@ weight: 500
 The Zendesk integration for Grafana OnCall handles ticket events sent from Zendesk webhooks.
 The integration provides grouping, auto-acknowledge and auto-resolve logic via customizable alert templates.
 
-## Configure Zendesk integration for Grafana OnCall
+> You must have an [Admin role]({{< relref "user-and-team-management" >}}) to create integrations in Grafana OnCall.
 
-You must have an Admin role to create integrations in Grafana OnCall.
+## Configuring Grafana OnCall to Receive Alerts from Zendesk
 
-1. In the **Integrations** tab, click **+ New integration to receive alerts**.
+1. In the **Integrations** tab, click **+ New integration**.
 2. Select **Zendesk** from the list of available integrations.
-3. Follow the instructions in the **How to connect** window to get your unique integration URL and review next steps.
+3. Enter a name and description for the integration, click **Create**
+4. The new page will open with the integration details. Copy the **OnCall Integration URL** from **HTTP Endpoint** section, you will need it when configuring Alertmanager.
```

Comment:
should this say 'Zendesk' instead of 'Alertmanager'?

```suggestion
4. A new page will open with the integration details. Copy the **OnCall Integration URL** from **HTTP Endpoint** section. You will need it when configuring Zendesk.
```